webpack.prod.config.js

```diff
@@ -62,6 +62,12 @@ module.exports = {
       hashDigest: 'hex',
       hashDigestLength: 20,
     }),
+    new DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('production'),
```

Comment:
Consider moving all environment variables to a single central storage, e.g. `docker-compose` files

Reply:
In this place we can't move it in other place because we strongly need a different configs for different environments.

Reply:
We need to maintain at least 3 different environments at all times:
1. Development
2. Testing
3. Production
It's already quite complex with multiple dockerfiles and .env files, so I'm all for reducing complexity when possible.
I suggest we leave it unchanged for now and get back to it in the future.